packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -70,6 +70,12 @@ function get_base_diffusers_model(model: ModelData): string {
 	return model.cardData?.base_model?.toString() ?? "fill-in-base-model";
 }
 
+function get_prompt_from_diffusers_model(model: ModelData): string {
+	return model.cardData?.widget?.[0]?.text?.toString()
```

Comment:
```suggestion
	return model.widgetData?.[0].text
```

I think

Reply:
yes @pcuenca is right, this is the right way to access this

Reply:
With `widgetData` it errors as it doesn't find the `text` property 
```
│ src/model-libraries-snippets.ts(75,26): error TS2339: Property 'text' does not exist on type 'WidgetExample'.
│   Property 'text' does not exist on type 'WidgetExampleChatInput<WidgetExampleOutput>'.
```

Reply:
for now you can do this:
```ts
(model.widgetData?.[0] as WidgetExampleTextInput).text
```

because the model card is for diffusers model, we can assume that the widgetData is for `text-to-image` pipeline which is `WidgetExampleTextInput`

Reply:
and you can remove https://github.com/huggingface/huggingface.js/blob/324e720c631ab871a920c371178e3d5dfdd7613b/packages/tasks/src/model-data.ts#L110-L115 since it is redundant to https://github.com/huggingface/huggingface.js/blob/324e720c631ab871a920c371178e3d5dfdd7613b/packages/tasks/src/model-data.ts#L92